app-editors/kakoune/kakoune-0_pre20161111.ebuild

```diff
@@ -27,7 +27,7 @@ DEPEND="
 	${RDEPEND}
 "
 
-PATCHES=( "${FILESDIR}/${PN}-makefile.patch" )
+PATCHES=( "${FILESDIR}/${PN}-0_pre20161111-makefile.patch" )
 
 pkg_pretend() {
```

Comment:
turn this into a function, and call it in `pkg_pretend` and `pkg_setup`. `pkg_pretend` is **not** the right place to check package invariants.